pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -63,12 +63,12 @@ func recognizers() []csrRecognizer {
 	recognizers := []csrRecognizer{
 		{
 			recognize:      isSelfNodeClientCert,
-			permission:     authorization.ResourceAttributes{Group: "certificates.k8s.io", Resource: "certificatesigningrequests", Verb: "create", Subresource: "selfnodeclient"},
+			permission:     authorization.ResourceAttributes{Group: "certificates.k8s.io", Resource: "certificatesigningrequests", Verb: "create", Subresource: "selfnodeclient", Version: util.AllVersion},
```

Comment:
I would inline `*` here instead of taking a package dependency, or declare the constant in the same package that declares the ResourceAttributes / AttributesRecord types